WinApi/User32/Helpers.cs

```diff
@@ -213,5 +220,63 @@ public static unsafe bool GetMonitorInfo(IntPtr hMonitor, out MonitorInfo lpmi)
             if (res) Rectangle.Subtract(ref lpRect, ref rc);
             return res;
         }
+
+        public static void PasteTextToClipboard(byte[] data)
+        {
+            if (User32Methods.OpenClipboard(new IntPtr()))
+            {
+                var bytesLength = data.Length;
+                var allocatedMemory = Marshal.AllocHGlobal(bytesLength);
+                try
+                {
+                    Marshal.Copy(data, 0, allocatedMemory, bytesLength);
+
+                    var result = User32Methods.SetClipboardData((uint)ClipboardFormat.CF_TEXT, allocatedMemory);
+
+                    if (result == IntPtr.Zero)
```

Comment:
I don't like the idea of throwing exceptions on a native API that indicates an error result. Instead, a better design would be to make it return a bool if IntPtr is zero, so the application can act in whichever way it deems necessary.